plugins/ui/src/deephaven/ui/components/picker.py

```diff
@@ -7,8 +7,16 @@
 from ..elements import BaseElement
 from .._internal.utils import create_props
 from ..types import ColumnName, Key
+from spectrum.layout import *
+from spectrum.validate import *
+from spectrum.events import *
```

Comment:
These are invalid imports. Should have the `.` in front of `spectrum` to indicate it's a relative directory:
```suggestion
from .spectrum.layout import *
from .spectrum.validate import *
from .spectrum.events import *
```